csrc/ir/nodes.cpp

```diff
@@ -4303,14 +4303,40 @@ std::string LinearOp::toInlineString(int indent_size) const {
 std::vector<PolymorphicValue> LinearOp::evaluate(
     const ExpressionEvaluator& ee,
     const std::vector<PolymorphicValue>& inputs) const {
-  const auto a = inputs.at(0).as<at::Tensor>();
-  const auto b = inputs.at(1).as<at::Tensor>();
+  const auto in = inputs.at(0).as<at::Tensor>();
```

Comment:
Can you add a comment about 1D weight support being temporarily removed until https://github.com/NVIDIA/Fuser/issues/2563 is resolved?

I've been trying to track all of them temp updates under this issue; granted, I've forgotten many times.